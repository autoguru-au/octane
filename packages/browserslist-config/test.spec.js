const browserslist = require('browserslist');
const config = require('.');

describe('config', () => {
	it('should resolve a list of browsers', () => {
		expect(browserslist(config).length).toBeGreaterThan(1);
	});

	it('should match snapshot', () => {
		expect(browserslist(config)).toMatchSnapshot();
	});
});
