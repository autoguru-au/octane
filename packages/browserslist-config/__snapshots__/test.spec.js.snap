// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config should match snapshot 1`] = `
Array [
  "and_chr 116",
  "and_ff 116",
  "chrome 116",
  "chrome 115",
  "chrome 114",
  "chrome 113",
  "chrome 109",
  "edge 116",
  "edge 115",
  "edge 114",
  "edge 113",
  "firefox 116",
  "firefox 115",
  "firefox 114",
  "ios_saf 16.5",
  "ios_saf 16.3",
  "ios_saf 16.2",
  "ios_saf 16.1",
  "ios_saf 16.0",
  "ios_saf 15.6",
  "ios_saf 12.2-12.5",
  "opera 101",
  "opera 100",
  "opera 99",
  "opera 98",
  "safari 16.5",
  "safari 16.4",
  "safari 16.3",
  "safari 16.2",
  "safari 15.6",
  "samsung 21",
  "samsung 20",
  "samsung 19.0",
  "samsung 18.0",
  "samsung 17.0",
  "samsung 16.0",
]
`;
